### PR TITLE
news107 touch-ups

### PR DESCRIPTION
**Pull request by jonatack**
This PR was motivated by seeing a missing period when reading today's newsletter, then added a few very optional suggestions.

### CODE REVIEW

File: _posts/en/newsletters/2020-07-22-newsletter.md [hunk 1]
```diff
@@ -127,10 +127,10 @@ release candidates.*
   sending of [multipath payments][topic multipath payments]---payments
   which are split into several parts, with each part routed using a
   different path.  In brief, the algorithm C-Lightning uses splits
-  a payment into parts approximately 0.0001 BTC in value (each part having its
+  a payment into parts of approximately 0.0001 BTC in value (each part having its
   amount randomly fuzzed by plus or minus 10%).  If any sent part fails,
   that part is split into two parts (roughly in half, plus or minus 10%)
-  and the two parts are resent.  The PR additionally adds a
+  and the two parts are re-sent.  The PR additionally adds a
```
Comment (jonatack) [inline _posts/en/newsletters/2020-07-22-newsletter.md:133]: I resentfully see that according to git grep, the homopath "resent" is used more often than "re-sent" in the writing, so happy to resentfully remove it :)

Comment (jonatack) [inline _posts/en/newsletters/2020-07-22-newsletter.md:133]: According to git grep, the homograph "resent" is used more often than "re-sent", so happy to (resentfully) remove it :)

**Comment by harding** [comment]
ACK, thanks!  (No strong opinion about re-sent versus resent.)

**Comment by jnewbery** [comment]
ACK. Thanks Jon!

**Pull request merged**